libs/documentation/src/lib/pages/layout/filter.service.ts

```diff
@@ -191,6 +191,14 @@ export class FilterService {
         ],
       },
     },
+    {
+      key: 'startDate',
+      type: 'datepicker',
+      templateOptions: {
+        hideOptional: true,
+        label: 'Date',
+      },
+    },
```

Comment:
For demo purpose, could you place this date filter inside either a pre-existing accordion group wrapper or have it be on its own accordion wrapper? The placement looks off in the de
<img width="525" alt="Screen Shot 2021-10-19 at 12 26 48 PM" src="https://user-images.githubusercontent.com/73653244/137952629-dab3bceb-075d-445d-a9dc-50a9ee4c6166.png">
mo